fifth_task/main.tf

```diff
@@ -0,0 +1,59 @@
+provider "aws" {
+  access_key = "AKIA2OB26KJBVD6VKJIZ"
+  secret_key = "q9zQb9QC2JaZSIUJjES/OVX9/kS4BfwlfRapTfzG"
```

Comment:
It is still a bad practice to commit access and secret keys in the repo :) Please consider using your `~/.aws/credentials` instead (default behaviour).